tests/containers/podman_netavark.pm

```diff
@@ -10,12 +10,16 @@
 use Mojo::Base 'containers::basetest';
 use testapi;
 use serial_terminal qw(select_serial_terminal);
-use version_utils qw(package_version_cmp is_transactional is_jeos is_leap is_sle_micro is_leap_micro is_sle);
+use version_utils qw(package_version_cmp is_transactional is_jeos is_leap is_sle_micro is_leap_micro is_sle is_microos);
 use containers::utils qw(get_podman_version registry_url);
 use transactional qw(trup_call check_reboot_changes);
 use utils qw(zypper_call);
 use Utils::Systemd qw(systemctl);
 
+sub is_cni_in_tw {
+   return (script_run("podman info -f '{{.Host.NetworkBackend}}' | grep cni") == 0) && is_microos && get_var('TDUP');
```

Comment:
```suggestion
   return (script_output("podman info -f '{{.Host.NetworkBackend}}'" =~ "cni") == 0) && is_microos && get_var('TDUP');
```
I would use `script_output` so we can actually see the output for debugging purposes.

Also I wonder why `get_var('TDUP')` is required? :thinking:

Reply:
`TDUP` is targeting the distro upgrade scenarios. In this case, there are 2 and only one of them is using an ancient image. Thus, this code might be removed in short time.

Reply:
fixed